src/multiinstances/internal/multiinstancesprovider.cpp

```diff
@@ -73,6 +83,28 @@ void MultiInstancesProvider::onMsg(const Msg& msg)
             mainWindow()->requestShowOnFront();
         }
     }
+    // Settings
+    else if (msg.type == MsgType::Request && msg.method == METHOD_PREFERENCES_IS_OPENED) {
+        bool isOpened = interactive()->isOpened(PREFERENCES_URI).val;
+        m_ipcChannel->response(METHOD_PREFERENCES_IS_OPENED, { QString::number(isOpened) }, msg.srcID);
+    } else if (msg.method == METHOD_SETTINGS_BEGIN_TRANSACTION) {
```

Comment:
Wrong key? Should be METHOD_ACTIVATE_WINDOW_WITH_OPENED_PREFERENCES, I think.

Reply:
big thanks, fixed, hmm, before making PR, I tested..., hmm